go/signedexchange/cmd/gen-signedexchange/main.go

```diff
@@ -195,6 +185,24 @@ func run() error {
 		return err
 	}
 
+	if !*flagIgnoreErrors {
+		certChain, err := certurl.NewCertChain(certs, []byte("dummy"), nil)
```

Comment:
Would you briefly summarize what the logic here is doing? (or factor out a `func` with a name which indicate what it is doing?